@xen-orchestra/lite/src/components/Tree.tsx

```diff
@@ -55,7 +55,7 @@ interface Props {
   //   }
   // ]
   collection: Array<ItemType>
-  defaultSelectedNodes?: Array<string>
+  defaultSelectedNode?: string
```

Comment:
IMHO, it's not necessary to modify Tree and TreeView components.

ping @pdonias